### PR TITLE
Grouping benchmarks

### PR DESCRIPTION
**Pull request by bmarkons**
Work regarding https://github.com/ruby-bench/ruby-bench-web/issues/232

### CODE REVIEW

File: app/views/admin/groups/_form.html.haml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+= form_for(group, url: admin_groups_path(group)) do |f|
+  - if group.errors.any?
+    .alert.alert-danger
+      %h2
+        = "#{pluralize(group.errors.count, "error")} prohibited this admin_group from being saved:"
```
Comment (robin850) [inline app/views/admin/groups/_form.html.haml]: s/admin_group/group ?

File: app/helpers/admin/groups_helper.rb [hunk 2]
```diff
@@ -0,0 +1,54 @@
+module Admin::GroupsHelper
+  def group_panel_html(group)
+    "
+    <div class='col-lg-6'>
```
Comment (bmarkons) [inline app/helpers/admin/groups_helper.rb]: :+1: definitely

File: app/helpers/admin/groups_helper.rb [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    group.benchmark_types.each do |benchmark_type|
+      benchmark_list << "<li>#{benchmark_type.category}</li>\n"
+    end
+
+    benchmark_list
+  end
+
+  def edit_link(group)
+    link_to(edit_admin_group_path(group), class: 'btn btn-default btn-circle') do
+      '<i class="fa fa-pencil"></i>'.html_safe
```
Comment (bmarkons) [inline app/helpers/admin/groups_helper.rb]: I was looking for this but without success. Thanks @robin850!

**Comment by bmarkons** [comment]
According to me, this one is ready to :ship: 

Please feel free to drop suggestions :)
@robin850 @noahgibbs

**Comment by noahgibbs** [comment]
Looks good to me.

**Comment by bmarkons** [comment]
@tgxworld May I merge this one?

File: app/controllers/admin/groups_controller.rb [hunk 4]
```diff
@@ -0,0 +1,63 @@
+class Admin::GroupsController < ApplicationController
```
Comment (robin850) [inline app/controllers/admin/groups_controller.rb]: Sorry, a bit late but why don't you rename the current `admin` controller to `admin/index` and create an `AdminController` class that would encapsulate the authentication logic, `set_admin`, etc. ? 

Maybe in another PR, I don't mind, just asking what you think about this. :-)

Comment (bmarkons) [inline app/controllers/admin/groups_controller.rb]: You are totally right. All admin controllers should inherit AdminController. This way I have duplicated stuff in AdminController and Admin::GroupsController.

> Maybe in another PR, I don't mind, just asking what you think about this. :-)

I wanted to do it in next PR, do you think it would be more proper doing it in this one?

Comment (robin850) [inline app/controllers/admin/groups_controller.rb]: It doesn't really matter, it will be in a separate commit anyway so it's up to you ! :-)
